pkg/webui/console/containers/device-onboarding-form/device-type-form-section/device-type-repository-form-section/device-card/connect.js

```diff
@@ -0,0 +1,27 @@
+// Copyright © 2021 The Things Network Foundation, The Things Industries B.V.
```

Comment:
```suggestion
// Copyright © 2022 The Things Network Foundation, The Things Industries B.V.
```
Please update on all files.